samples/MvcSubAreaSample.Web/Areas/Restaurant/Areas/Hours/Controllers/HomeController.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace MvcSubAreaSample.Web.Areas.Restaurant.Hours.Controllers
```

Comment:
My philosophy is to just write code in a sample the way a user would write it. Everything you see in a sample that you don't immediately grok has a mental cost associated.

I tend to avoid using things like `.Controllers` in namespaces in general. Using the area/subarea in the namespace seems pretty usual, since areas are really about code organization.